lib/permutations.rb

```diff
@@ -1,4 +1,21 @@
 
 def permutations?(string1, string2)
-  
+  l1 = string1.length
+  l2 = string2.length
+  return false if l1 != l2
```

Comment:
These variables are a little hard to follow because they don't describe their values. Alternatives could be:

```ruby
  string1_length = string1.length
  string2_length = string2.length
```

Longer, descriptive variable names are preferable to short, cryptic ones :-)